samples/dnn/dasiamrpn_tracker.py

```diff
@@ -195,23 +211,19 @@ def __get_subwindow_tracking(self, im, model_size, original_sz):
 
         if not np.array_equal(model_size, original_sz):
             im_patch_original = cv.resize(im_patch_original, (model_size, model_size))
-
         return im_patch_original
 
-#function for reading paths, bounding box drawing, showing results
+# Sample for using DaSiamRPN tracker
 def main():
     parser = argparse.ArgumentParser(description="Run tracker")
-    parser.add_argument("--net", type=str, default="dasiamrpn_model.onnx", help="Full path to onnx model of net")
-    parser.add_argument("--kernel_r1", type=str, default="dasiamrpn_kernel_r1.onnx", help="Full path to onnx model of kernel_r1")
-    parser.add_argument("--kernel_cls1", type=str, default="dasiamrpn_kernel_cls1.onnx", help="Full path to onnx model of kernel_cls1")
```

Comment:
Why these arguments have been removed? We need to let user control the paths.

Reply:
I moved it into parameters of the constructor - we can write paths during creating tracker.

Reply:
1. That should be done for users who wants to run a script but cannot modify it or don't know how to do it
2. We have these flags in initial version so it'll be a regression

Reply:
I understand that you are talking about the sample. But I want to change class for further more convenient use as an independent class. As an example, I used GOTURN from opencv-contrib- all models paths inside the tracker, not in the sample.
Maybe it is a regression for one specific sample, but IMHO, it is better for the tracker.

Reply:
I don't see any problem to have both class level API and sample level API:
```python
class DaSiamRPNTracker:
    # Initialization of used values, initial bounding box, used network
    def __init__(self, net="dasiamrpn_model.onnx", kernel_r1="dasiamrpn_kernel_r1.onnx", kernel_cls1="dasiamrpn_kernel_cls1.onnx"):
```
```python
def main():
    parser = argparse.ArgumentParser(description="Run tracker")
    parser.add_argument("--net", type=str, default="dasiamrpn_model.onnx", help="Full path to onnx model of net")
    parser.add_argument("--kernel_r1", type=str, default="dasiamrpn_kernel_r1.onnx", help="Full path to onnx model of kernel_r1")
    parser.add_argument("--kernel_cls1", type=str, default="dasiamrpn_kernel_cls1.onnx", help="Full path to onnx model of kernel_cls1")

tracker = DaSiamRPNTracker(args.net, args.kernel_r1, args.cls1)
```

Reply:
OK.
Sorry, I misunderstood your proposals.